data/functions/itemized.sql

```diff
@@ -16,6 +16,28 @@ END
 $$ LANGUAGE plpgsql;
 
 
+CREATE OR REPLACE FUNCTION add_partition_cycles(start_year NUMERIC, amount NUMERIC) RETURNS VOID AS $$
+DECLARE
+    end_year NUMERIC = start_year + (amount - 1) * 2;
+    master_table_name TEXT;
+    child_table_name TEXT;
+    schedule TEXT;
```

Comment:
One more small thing with this, @vrajmohan:  If the start year given is something like `2015` we want to make sure that it is handled appropriately, which means grabbing the cycle year based on the user input.  So in the case of `2015`, that should become `2016`, which is the cycle end year.

When I used `2015` locally, it created another child table called `ofec_sched_a_2014_2015`, which should not exist as that is not a valid cycle.

We have a [`get_cycle`](https://github.com/18F/openFEC/blob/develop/data/functions/utils.sql#L1) method in the SQL already, it should just be a matter of calling that with the `start_year` argument given and then using that returned value as the `end_year` and using `cycle - 1` as the start year for creating each child table.

Does this make sense?  Happy to hop on a quick chat if not!